src/coreclr/pal/tests/palsuite/eventprovider/CMakeLists.txt

```diff
@@ -6,10 +6,10 @@ set(SOURCES
 set(EVENT_MANIFEST ${VM_DIR}/ClrEtwAll.man)
 set(TEST_GENERATOR ${CLR_DIR}/scripts/genEventingTests.py)
 
-include(FindPythonInterp)
+include(FindPython)
```

Comment:
Shouldn't that have been `find_package(Python)`?

Reply:
@teo-tsirpanis we're using `include(FindPython)` in the rest of the codebase so I used it here as well. Apparently there is no real difference though it looks like `find_package` is preferred: https://stackoverflow.com/a/72694177